src/main/java/cz/scholz/strimzi/api/examples/cruisecontrol/CreateRebalance.java

```diff
@@ -0,0 +1,42 @@
+package cz.scholz.strimzi.api.examples.cruisecontrol;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClientBuilder;
+import io.strimzi.api.kafka.Crds;
+import io.strimzi.api.kafka.model.KafkaRebalance;
+import io.strimzi.api.kafka.model.KafkaRebalanceBuilder;
+import io.strimzi.api.kafka.model.balancing.KafkaRebalanceState;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+
+
+public class CreateRebalance {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CreateRebalance.class);
+    private static final String NAMESPACE = "myproject";
+    private static final String REBALANCE_NAME = "myrebalance";
+
+    public static void main(String[] args) {
+        try (KubernetesClient client = new KubernetesClientBuilder().build()) {
+            LOGGER.info("create rebalance");
+            KafkaRebalanceBuilder kafkaRebalanceBuilder = new KafkaRebalanceBuilder().editMetadata()
```

Comment:
Since it is a new resource, maybe it should use `withNewMetadata()` instead of `editMetadata`?

Reply:
done